### PR TITLE
fix(sum): fix getSum loop when component is disable

### PR DESCRIPTION
**Pull request by juanlopez1**

### CODE REVIEW

File: src/components/Sum/index.js [hunk 1]
```diff
@@ -2,61 +2,60 @@ import React from 'react';
...
-import {filter, toNumber, isNil, sum, isNaN} from 'lodash';
+import {isEmpty, filter, toNumber, isNil, sum} from 'lodash';
 
 import TextWithBadge from '../TextWithBadge';
 import {handleChange} from '../../util';
+import commonStyles from '../commonStyles';
 import styles from './styles';
 
-const getTotal = (section, question, onChange) => {
+const getTotal = (section, {name, fieldsToAdd}, onChange) => {
```
Comment (navarroaxel) [inline src/components/Sum/index.js]: rename to `getSum`

File: src/components/Sum/index.js [hunk 2]
```diff
@@ -2,61 +2,60 @@ import React from 'react';
...
     const addends = filter(
-        question.fieldsToAdd,
+        fieldsToAdd,
         field => !isNil(section[field])
     ).map(
         field => toNumber(section[field])
     );
+    const total = sum(addends);
```
Comment (navarroaxel) [inline src/components/Sum/index.js]: rename to `result`

File: src/components/Sum/index.js [hunk 3]
```diff
@@ -2,61 +2,60 @@ import React from 'react';
...
-        <View style={computedStyles.container}>
+        <View style={disabled ? commonStyles.disabledContainer : computedStyles.container}>
             {question.text && <TextWithBadge
                 question={question}
                 style={textWithBadgeStyle}
             />}
-            <Text style={computedStyles.totalLabel}>
-                {getTotal(section, question, onChange)}
+            <Text style={computedStyles.sumLabel}>
+                {!disabled && getTotal(section, question, onChange)}
```
Comment (navarroaxel) [inline src/components/Sum/index.js]: xq el `!disabled` no se le aplica al `Text` component?

File: src/components/Sum/index.js (hunk 1 above)
Comment (juanlopez1) [reply src/components/Sum/index.js]: Listo

File: src/components/Sum/index.js (hunk 2 above)
Comment (juanlopez1) [reply src/components/Sum/index.js]: Listo

File: src/components/Sum/index.js (hunk 3 above)
Comment (juanlopez1) [reply src/components/Sum/index.js]: Listo

File: src/components/Sum/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (juanlopez1) [reply src/components/Sum/index.js]: Listo

**Pull request merged**